sunpy/visualization/animator/image.py

```diff
@@ -159,12 +160,12 @@ class ImageAnimatorWCS(ImageAnimator):
     """
     def __init__(self, data, wcs=None, image_axes=[-1, -2], unit_x_axis=None, unit_y_axis=None,
                  axis_ranges=None, **kwargs):
-        if not isinstance(wcs, astropy.wcs.WCS):
+        if not isinstance(wcs, (astropy.wcs.WCS, SlicedLowLevelWCS)):
```

Comment:
This should just be checking for isinstance of the low level base class.

Reply:
Sure!